basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,182 @@
 const helpers = require('./project-1');
 
 // start testing!
+// are equal
+describe( 'areEqual(x, y)', function ()
```

Comment:
Consider checking the Type of return value to be Boolean, and perhaps checking arrays and objects against each other.